packages/client-python/src/rocketride/mixins/dashboard.py

```
@@ -0,0 +1,81 @@
+
+        Example:
+            dashboard = await client.get_dashboard()
+            for task in dashboard['tasks']:
+                print(f"{task['id']}: {task['status']} ({task['elapsedTime']:.0f}s)")
+        """
+        request = self.build_request(command='rrext_dashboard', arguments={})
+        response = await self.request(request)
+
+        if self.did_fail(response):
+            error_msg = response.get('message', 'Failed to retrieve dashboard')
+            raise RuntimeError(f'Failed to retrieve dashboard: {error_msg}')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)